include/bluetooth/audio/vcs.h

```diff
@@ -0,0 +1,504 @@
+ *                      value.
+ * @param inst          Pointer to the Volume Offset Control Service instance.
+ * @param description   The description to set. Value will be copied.
+ *
+ * @return 0 if success, errno on failure.
+ */
+int bt_vcs_vocs_description_set(struct bt_conn *conn, struct bt_vocs *inst,
+				const char *description);
+
+/**
+ * @brief Deactivates a Audio Input Control Service instance.
+ *
+ * Audio Input Control Services are activated by default, but this will allow
+ * the server deactivate a Audio Input Control Service.
+ *
+ * @param inst   Pointer to the Audio Input Control Service instance.
+ *
+ * @return 0 if success, errno on failure.
+ */
+int bt_vcs_aics_deactivate(struct bt_aics *inst);
+
+/**
+ * @brief Activates a Audio Input Control Service instance.
+ *
+ * Audio Input Control Services are activated by default, but this will allow
+ * the server reactivate a Audio Input Control Service instance after it has
+ * been deactivated with @ref bt_vcs_aics_deactivate.
+ *
+ * @param inst   Pointer to the Audio Input Control Service instance.
+ *
+ * @return 0 if success, errno on failure.
+ */
+int bt_vcs_aics_activate(struct bt_aics *inst);
+
+/**
+ * @brief Read the Audio Input Control Service input state.
+ *
+ * @param conn   Connection to peer device, or NULL to read local server value.
+ * @param inst   Pointer to the Audio Input Control Service instance.
+ *
+ * @return 0 if success, errno on failure.
+ */
+int bt_vcs_aics_state_get(struct bt_conn *conn, struct bt_aics *inst);
+
+/**
+ * @brief Read the Audio Input Control Service gain settings.
+ *
+ * @param conn   Connection to peer device, or NULL to read local server value.
+ * @param inst   Pointer to the Audio Input Control Service instance.
+ *
+ * @return 0 if success, errno on failure.
+ */
+int bt_vcs_aics_gain_setting_get(struct bt_conn *conn, struct bt_aics *inst);
+
+/**
+ * @brief Read the Audio Input Control Service input type.
+ *
+ * @param conn   Connection to peer device, or NULL to read local server value.
+ * @param inst   Pointer to the Audio Input Control Service instance.
+ *
+ * @return 0 if success, errno on failure.
+ */
+int bt_vcs_aics_type_get(struct bt_conn *conn, struct bt_aics *inst);
+
+/**
+ * @brief Read the Audio Input Control Service input status.
+ *
+ * @param conn   Connection to peer device, or NULL to read local server value.
+ * @param inst   Pointer to the Audio Input Control Service instance.
+ *
+ * @return 0 if success, errno on failure.
+ */
+int bt_vcs_aics_status_get(struct bt_conn *conn, struct bt_aics *inst);
+
+/**
+ * @brief Mute the Audio Input Control Service input.
+ *
+ * @param conn   Connection to peer device, or NULL to read local server value.
+ * @param inst   Pointer to the Audio Input Control Service instance.
+ *
+ * @return 0 if success, errno on failure.
+ */
+int bt_vcs_aics_unmute(struct bt_conn *conn, struct bt_aics *inst);
```

Comment:
Documentation says "mute", function says "unmute"

Reply:
Hah, I think we had that same comment for AICS, right? 
Will fix of course :)

Reply:
It was familiar, yes.